front/lib/api/assistant/configuration.ts

```diff
@@ -383,6 +384,13 @@ async function fetchWorkspaceAgentConfigurationsForView(
           },
         }).then(groupByAgentConfigurationId)
       : Promise.resolve({} as Record<number, AgentProcessConfiguration[]>),
+    variant === "full"
```

Comment:
Is it time to refactor into a generic "for all action types" rather? (to avoid the same omission that led to this fix)

Can do it directly here or in a follow up PR. 
WDYT?